docs/intro.rst

```diff
@@ -26,7 +26,7 @@ It's recommended you set up your D-Wave Cloud Client configuration through the
 
 As described in the :std:doc:`Using a D-Wave System <oceandocs:overview/dwavesys>` section
 of Ocean Documentation, for your code to access remote D-Wave compute resources, you must
-configure communication through SAPI; for example, your code needs the SAPI URL and your API
+configure communication through SAPI; for example, your code needs your API
```

Comment:
Should we mention here that API endpoint URL can be changed, to accommodate for example on premise installs of QCs?

Reply:
This is very much the entry point to this tool, and I'm thinking that for now the number of users who might benefit from that info is small compared to those for whom it will be a distraction.